lib/components/T3Form/T3FormFieldList/T3FormFieldList.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <div>
+    <div
+      v-for="(field, keyItem) in elements"
+      :key="keyItem"
+      :class="[`t3-form-field-${field.identifier}`, getCssClass(field, classes)]"
+    >
+      <template v-if="field.type === 'Fieldset'">
+        <T3FormFieldset :field="field">
+          <T3FormFieldList
+            ref="nested"
+            :elements="field.elements"
+            :components="components"
+            :classes="classes"
+            @change="updateModel"
+          />
+        </T3FormFieldset>
+      </template>
+      <template v-else>
+        <component
+          :is="getComponentField(field)"
+          v-model="model[field.name]"
+          :field="field"
+        />
+      </template>
+    </div>
+  </div>
+</template>
+<script>
+import Vue from 'vue'
+import { ValidationProvider, extend } from 'vee-validate'
+import extendableComponents from '../../../mixins/component/extendableComponents'
+import T3FormFieldset from '../T3FormFieldset/T3FormFieldset.vue'
+import T3FormField from '../T3FormField/T3FormField.vue'
+import T3FormFieldHidden from '../T3FormFieldHidden/T3FormFieldHidden.vue'
+import T3FormFieldSelect from '../T3FormFieldSelect/T3FormFieldSelect.vue'
+import * as validators from '../T3FormValidators'
+
+export default Vue.extend({
+  name: 'T3FormFieldList',
+  components: {
+    ValidationProvider,
+    T3FormFieldset,
+    T3FormField,
+    T3FormFieldHidden,
+    T3FormFieldSelect,
+    T3FormFieldSingleSelect: T3FormFieldSelect
+  },
+  mixins: [extendableComponents],
+  props: {
+    elements: {
+      type: Array,
+      required: true
+    },
+    components: {
+      type: [Object, Boolean],
+      required: false,
+      default: false
+    },
+    classes: {
+      type: [Object, Boolean],
+      required: false,
+      default: false
+    },
+    extendRules: {
+      type: Array,
+      default: () => []
+    },
+    extendTypes: {
+      type: Object,
+      default: () => ({})
+    }
+  },
+  data () {
+    return {
+      rules: [],
+      availableCustomTypes: {
+        Honeypot: 'T3FormFieldHidden'
+      },
+      model: {}
+    }
+  },
+  computed: {
+    customTypes () {
+      return ({
+        ...this.availableCustomTypes,
+        ...this.extendTypes
+      })
+    }
+  },
+  watch: {
+    model: {
+      handler (newModel) {
+        this.$emit('change', Object.assign(this.model, newModel))
+      },
+      deep: true
+    }
+  },
+  beforeMount () {
```

Comment:
I didn't see this hook for a long time :P. Are there any contraindications to putting this setup inside `created`?